docs/diagrams/BetterModelClassDiagram.puml

```diff
@@ -4,18 +4,19 @@ skinparam arrowThickness 1.1
 skinparam arrowColor MODEL_COLOR
 skinparam classBackgroundColor MODEL_COLOR
 
+AddressBook *-right-> "1" UniqueReviewList
 AddressBook *-right-> "1" UniquePersonList
```

Comment:
Remember to remove the UniquePersonList.

Reply:
Removed in latest push